.travis.yml

```diff
@@ -0,0 +1,28 @@
+dist: trusty
+language: rust
+rust:
+  - 1.19.0
+  - nightly
```

Comment:
Currently we only build with Nightly rust. Is it possible to only run the tests for the Nightly configuration, at least for now? Is it as simple as just removing the 1.19.0 line here?